test/cockroachdb_test.go

```diff
@@ -0,0 +1,82 @@
+package helm_testing
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/require"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+func Test_Role(t *testing.T) {
```

Comment:
The below tests should be broken out into individual tests please, rather than rolled all together.

Reply:
I think `t.Run` is fine, since this test is focused on rendering and validating the `role.yaml` template. I think we could also potentially use [table-driven tests](https://github.com/golang/go/wiki/TableDrivenTests) to make this even more idiomatic. WDYT?